### PR TITLE
elasticsearch @6.1.2: new port

### PR DESCRIPTION
**Pull request by GerardSoleCa**
#### Description
Elasticsearch is an open-source, RESTful, distributed search and analytics engine built on Apache Lucene

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.12.6 16G1036
Xcode 8.3.3 8E3004b

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by GerardSoleCa** [comment]
Hi, I know its an obsolete version of elasticsearch, but some people require this version due to legacy "business".
My idea, is to update to current stable onces aproved, so we can select the old or new version when installing this port.

Is there an easy way to admit both ports? Doing a future pull request/commit with the upgraded version will allow us to support this usecase?

**Comment by mf2k** [comment]
This is not the proper way in Macports to support an older version. Honestly, I don't think you are going to get much support from the Macports team for an old version of this port with no maintainer. So I think your options are:
 - Agree to maintain both ports
 - Just submit for the latest version

If you are willing to maintain these, then update your portfile to use the latest version and then add a sub-port for the older version and call it elasticsearch176 or similar. Otherwise just do the same but leave out the sub-port section for the old version.

**Comment by GerardSoleCa** [comment]
Hi @mf2k,
Following your recommendations, I've updated the Portfile to the latest version of elasticsearch (v6.1.1), I hope this suits the macport submitting guidelines :)

At this point, I'm not able to commit my self as a contributor to this port, as it's not my day to day software. So, in this way, anyone can take the ownership of it if they want :)

I hope everything is ok now, thanks for your reply!

**Comment by pmetzger** [comment]
@mf2k Should we just commit this at this point?

File: databases/elasticsearch/Portfile [hunk 1]
```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                elasticsearch
+version             6.1.1
+revision            1
```
Comment (grimreaper) [inline databases/elasticsearch/Portfile]: revision not needed for a new port

File: databases/elasticsearch/Portfile [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+categories          databases java
+maintainers         nomaintainer
```
Comment (grimreaper) [inline databases/elasticsearch/Portfile]: new ports should have a maintainer

File: databases/elasticsearch/Portfile [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+                    Elasticsearch requires Java, Java version can be up to Java 8. \
+                    Java 9 is not supported and Elasticsearch won't work.
+
+platforms           darwin
+supported_archs     noarch
+license             Apache-2
+
+homepage            https://www.elastic.co/
+master_sites        https://artifacts.elastic.co/downloads/elasticsearch/
+distname            ${name}-${version}
```
Comment (grimreaper) [inline databases/elasticsearch/Portfile]: isn't this default?

File: databases/elasticsearch/Portfile [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+distfiles           ${distname}.zip
+checksums           rmd160 cc4bd477503e276d7f075eb26a784450627328fc \
```
Comment (grimreaper) [inline databases/elasticsearch/Portfile]: include size too if you can

File: databases/elasticsearch/Portfile (hunk 2 above)
Comment (pmetzger) [reply databases/elasticsearch/Portfile]: The OP said he can't commit to being the maintainer, though he could always put himself and "openmaintainer" down.

File: databases/elasticsearch/Portfile (hunk 1 above)
Comment (pmetzger) [reply databases/elasticsearch/Portfile]: Yah, @GerardSoleCa, please remove `revision`.

**Comment by pmetzger** [comment]
@grimreaper ?

**Comment by GerardSoleCa** [comment]
Hi,
I updated the Portfile with your recommendations. Let me reword the last commit, as it updates elasticsearch's version two 6.1.2, and I missed to change the name.

I'll do it today :)

**Comment by pmetzger** [comment]
Merged! Thank you for your contribution to MacPorts!

**Pull request merged**